libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -1328,3 +1334,11 @@ void AP_GPS_UBLOX::Write_DataFlash_Log_Startup_messages() const
                                            _version.swVersion);
     }
 }
+
+void AP_GPS_UBLOX::_check_new_itow(uint32_t itow)
```

Comment:
Can we lift this to `AP_GPS_Backend`? A couple of the other drivers should use the same logic. (IE every SBF message includes a TOW), we'd have to pass the _payload_length + header size in then, but it lets use this in common). That's not a hold up on merging this though.